### PR TITLE
JSDK-2951 Adding in tests for localAudioTrack methods

### PR DESCRIPTION
**Pull request by PikaJoyce**
Adding in some quick tests and updating changelog to prepare for TS definition release
- [x] Adding in addition compilation tests for `LocalAudioTrack`
- [x] Beef up CHANGELOG.md entry to include examples of how to import TS definitions into projects, and a note about definitely typed
- [x] Added in fixes for [JSDK-3131](https://issues.corp.twilio.com/browse/JSDK-3131)

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: tsdef/twilio-video-tests.ts [hunk 1]
```diff
@@ -19,6 +19,9 @@ function getAudioTrack(track: Video.LocalAudioTrack) {
   const localAudioTrack = track;
   localAudioTrack.attach();
   localAudioTrack.detach();
+  localAudioTrack.disable();
+  localAudioTrack.enable();
+  localAudioTrack.stop();
```
Comment (manjeshbhargav) [inline tsdef/twilio-video-tests.ts:24]: @PikaJoyce Please store the return value in a variable of type `LocalAudioTrack` (`LocalVideoTrack` in video tests). The objective is to make sure we are returning the proper video types.

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,10 +8,18 @@ The Twilio Programmable Video SDKs use [Semantic Versioning](http://www.semver.o
 New Features
 ------------
 
-- You can now import type definitions for the SDK APIs to your project as shown below. (JSDK-3007)
+- You can now import type definitions for the SDK APIs to your project as shown below. Twilio-Video types will take priority when installed at the same time as the Definitely Typed type definitions. (JSDK-3007)
```
Comment (manjeshbhargav) [inline CHANGELOG.md:11]:
```suggestion
- You can now import type definitions for the SDK APIs to your project. These type definitions will take precedence over any other type definitions that you may be using, like `@types/twilio-video` from DefinitelyTyped. (JSDK-3007)
```

Comment (makarandp0) [inline CHANGELOG.md:11]: I suggest something like:
twilio-video now includes typescript definitions for the SDK APIs. Previously typescript developers relied on [definitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/1a7a99db8ec25d48f3dfec146af742e5bc40a5f7/types/twilio-video/index.d.ts) for these definitions. We would like to thank folks at DefinitelyTyped for maintaining these definitions. Going forward these will be included in the library and will take precedence over typescript definitions. You can now remove definitelyTyped dependency from your project.

File: tsdef/twilio-video-tests.ts (hunk 1 above)
Comment (makarandp0) [inline tsdef/twilio-video-tests.ts:24]:
```suggestion
  localAudioTrack.disable().enable().stop();
```

**Review by makarandp0 — APPROVED** [review]
we should see if  @donaltoomey can take a look at the changelog as well.

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,10 +8,18 @@ The Twilio Programmable Video SDKs use [Semantic Versioning](http://www.semver.o
...
-    import * as Video from 'twilio-video';
+  import * as Video from 'twilio-video';
+
+  let room: Video.Room | null = null;
+
+  async function connectToRoom() {
+    room = await Video.connect('$TOKEN', { name: 'room-name' });
+    console.log('Connected to Room "%s"', room.name);
+    return room;
+  }
```
Comment (manjeshbhargav) [inline CHANGELOG.md:22]:
```suggestion
  import * as Video from 'twilio-video';
\`\`\`

You can access the types of the public API classes from the `Video` namespace as shown below:

\`\`\`
Video.connect('token', { name: 'my-cool-room' }).then((room: Video.Room) => {
  console.log('Connected to Room:', room.name);
  room.on('participantConnected', (participant: Video.RemoteParticipant) => {
    console.log('RemoteParticipant joined:', participant.identity);
  });
});
\`\`\`
```

**Review by manjeshbhargav — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (PikaJoyce) [reply CHANGELOG.md:11]: @makarandp0 I like this! @manjeshbhargav Should I also include an acknowledgement to DefinitelyTyped in the changelog?

**Review by donaltoomey — APPROVED** [review]
LGTM

**Review by makarandp0 — APPROVED** [review]

**Review by manjeshbhargav — APPROVED** [review]

**Pull request merged**